lib/ui/window.dart

```diff
@@ -871,3 +871,31 @@ class FrameData {
   /// If not provided, defaults to -1.
   final int frameNumber;
 }
+
+/// Platform specific configuration for gesture behavior, such as touch slop.
+class GestureSettings {
```

Comment:
When we land this we should add some more details to the docs, e.g. describing who is expected to create instances of this, where they're expected to be used, pointing people to MediaQuery, etc.